packages/cli/src/ui/components/SuggestionsDisplay.tsx

```
@@ -7,8 +7,8 @@
 import { Box, Text } from 'ink';
 import { theme } from '../semantic-colors.js';
 import { PrepareLabel, MAX_WIDTH } from './PrepareLabel.js';
-import { CommandKind } from '../commands/types.js';
-import { Colors } from '../colors.js';
+import { SuggestionsDisplay, MAX_WIDTH } from './SuggestionsDisplay.js';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'MAX_WIDTH'. Consider to delete it or rename it.

'MAX_WIDTH' is defined here:


(lint/suspicious/noRedeclare)